service/lib/dinstaller/dbus/software_service.rb

```diff
@@ -71,10 +72,18 @@ def service
       # @return [Array<::DBus::Object>]
       def dbus_objects
         @dbus_objects ||= [
-          DInstaller::DBus::Software::Manager.new(@backend, logger),
+          dbus_software_manager,
           DInstaller::DBus::Software::Proposal.new(logger)
         ]
       end
+
+      # @return [DInstaller::DBus::Software::Manager]
+      def dbus_software_manager
+        return @dbus_software_manager if @dbus_software_manager
```

Comment:
NP: I guess this could simply be `@dbus_software_manager ||= DInstaller::DBus::Software::Manager.new(backend, logger)`

Reply:
Yes, you are right, it was a leftover.